src/main/scala/rocket/HellaCache.scala

```diff
@@ -51,8 +40,8 @@ trait HasL1HellaCacheParameters extends HasCacheParameters with HasCoreParameter
 
   require(isPow2(nSets))
   require(rowBits >= coreDataBits)
-  require(rowBits <= outerDataBits)
-  require(xLen <= outerDataBits) // would need offset addr for puts if data width < xlen
+  require(rowBits == cacheDataBits) // TODO should rowBits even be seperably specifiable?
```

Comment:
I think we can agree that we always want to be able to sink an entire beat of refilled data per cycle, so it seems like TileLink width is at least a lower bound on rowBits. I'd go further and claim that the refill width is the broader architectural parameter; picking a set of RAMs of whatever optimal width so as to fulfill the requirement is backend flow decision that shouldn't even be expressed in Chisel?